pytest_tests/testsuites/object/test_object_lock.py

```diff
@@ -144,6 +157,61 @@ def new_locked_storage_object(
 
         return storage_object
 
+    @pytest.fixture(scope="function")
+    @allure.title("Enable metabase resync on start")
+    def enable_metabase_resync_on_start(self):
+        """
+        If there were already any environment variables in the DOCKER_COMPOSE_ENV_FILE, they should be retained and
+        NEOFS_STORAGE_SHARD_0_RESYNC_METABASE and NEOFS_STORAGE_SHARD_1_RESYNC_METABASE should be added to the file.
+
+        If NEOFS_STORAGE_SHARD_0_RESYNC_METABASE and NEOFS_STORAGE_SHARD_1_RESYNC_METABASE are explicitly specified
+        as false, they must be changed to true.
+
+        If DOCKER_COMPOSE_ENV_FILE is empty, NEOFS_STORAGE_SHARD_0_RESYNC_METABASE and
+        NEOFS_STORAGE_SHARD_1_RESYNC_METABASE must be added to DOCKER_COMPOSE_ENV_FILE.
+
+        Of course, after the test, the DOCKER_COMPOSE_ENV_FILE must return to its initial state.
+        """
+        file_path = DOCKER_COMPOSE_ENV_FILE
+        if not os.path.exists(file_path):
+            pytest.fail(f'File {file_path} does not exist!')
+
+        with open(file_path, 'r') as file:
+            lines = file.readlines()
+        logger.debug(f"Initial file content:\n{''.join(lines)}")
+
+        replacements = {
+            'NEOFS_STORAGE_SHARD_0_RESYNC_METABASE=false': 'NEOFS_STORAGE_SHARD_0_RESYNC_METABASE=true\n',
+            'NEOFS_STORAGE_SHARD_1_RESYNC_METABASE=false': 'NEOFS_STORAGE_SHARD_1_RESYNC_METABASE=true\n'
+        }
+
+        unprocessed_lines = set(replacements.values())
+
+        modified_lines = []
+
+        for line in lines:
```

Comment:
well, since that is a tricky hack, isn't appending the file with the required envs an easier approach? or renaming the original file, reading it, creating a new file (and appending the needed lines to it), and then renaming it back?

but that is just a reading simplification. mb you think differently

Reply:
> well, since that is a tricky hack, isn't appending the file with the required envs an easier approach? or renaming the original file, reading it, creating a new file (and appending the needed lines to it), and then renaming it back?
> 
> but that is just a reading simplification. mb you think differently

Yeah, I've been thinking about it.
However, it turns out that I have to copy the docker-compose.yml file from dev-env and change the path to the new file with new environment variables.
It seems to me that this will complicate the code too much and make it incomprehensible.